flight/targets/sprf3e/bl/pios_board.c

```diff
@@ -0,0 +1,88 @@
+/**
+ ******************************************************************************
+ * @addtogroup Bootloader Bootloaders
+ * @{
+ * @addtogroup spf3e SP Racing F3E support files
```

Comment:
These need to be consistent through all the files. The first word is the group identifier used internally in Doxygen to link all the group members (technically you can define the group title once and just add things with the identifier but that doesn't really work for us because we don't have a central place to define all the groups), and the remainder is the title shown in the documentation, including the navigation tree. i.e.
![image](https://cloud.githubusercontent.com/assets/9995998/24826956/f5f03b84-1c95-11e7-954f-f2366dfd7d1e.png)
"SP Racing F3E support files" is way too long for there IMO. We have used uppercase identifiers through the rest of the boards (IIRC).

Can you please make them all ` * @addtogroup SPRF3 SP Racing F3 Evo`